### DISCUSSION TITLE
syslog-ng Python packaging

### DISCUSSION THREAD

**Discussion by czanik** [syslog-ng/syslog-ng#4481] [General]
In my latest blog I described some syslog-ng Python packaging related problems, and some of my plans fixing those: [syslog-ng Python packaging](https://www.syslog-ng.com/community/b/blog/posts/syslog-ng-python-packaging) 

Here I'd like to ask for your feedback, if you have any problems, suggestions, etc., what I should try to address in my openSUSE / SLES / fedora / RHEL / FreeBSD packaging.

**Comment by erempel** [comment]
I like you proposed plan of splitting the different drivers into their own packages with their own set of dependencies. This allows the use of packages from other repositories or locally build repositories to satisfy the requirements of the needed drivers, while not having to satisfy requirement for unneeded drivers.
